hints_tests.py

```diff
@@ -12,11 +12,16 @@ def test_program(program_name: str):
     test_program("hint_print_vars")
     test_program("vm_scope_hints")
     test_program("is_le_felt_hint")
-    test_program("assert_le_felt_hint")
-    test_program("is_nn_hint")
     test_program("ec_mul_inner")
-    # test_program("ec_negate")
+    # test_program("ec_negate") # FAILING (Issue with ids structs)
```

Comment:
This should now be working.

Reply:
Yes, now is failling due to:
`ValueError: Variable value not present in current execution scope`